std_msgs/Header header

# Number of features after each outlier removal step.
int16 before_tracking
int16 after_tracking
int16 after_matching
int16 after_ransac
